labelbox/client.py

```diff
@@ -658,20 +676,23 @@ def create_project(self, **kwargs) -> Project:
                 "Creating a project without specifying media_type"
                 " through this method will soon no longer be supported.")
 
-        if not queue_mode:
-            logger.warning(
-                "Default createProject behavior will soon be adjusted to prefer "
-                "batch projects. Pass in `queue_mode` parameter explicitly to opt-out for the "
-                "time being.")
-        elif queue_mode == QueueMode.Dataset:
-            logger.warning(
-                "QueueMode.Dataset will eventually be deprecated, and is no longer "
-                "recommended for new projects. Prefer QueueMode.Batch instead.")
+        quality_mode = kwargs.get("quality_mode")
```

Comment:
Looks like dupe lines, since we do the same line 684-..

Reply:
These aren't duplicate lines; one exists to inform the user that a default has been selected, and the other exists to translate the given quality_mode to something our backend can interpret